docs/README.md

```diff
@@ -1,147 +1,9 @@
-<!-- @todo: https://github.com/badges/shields/pull/7759 -->
-![GitHub branch checks state](https://img.shields.io/github/checks-status/GoogleCloudPlatform/bank-of-anthos/main)
-[![Website](https://img.shields.io/website?label=live%20demo&url=https%3A%2F%2Fbank-of-anthos.xyz%2F)](https://bank-of-anthos.xyz)
+# Documentation
 
-# Bank of Anthos
+<!-- This section is duplicated in the root README -->
```

Comment:
nit: Maybe reference the root README here?

Reply:
Fixed!